lib/hushed/blackboard.rb

```diff
@@ -40,5 +42,40 @@ def build_document(type, contents)
       end
       namespace.const_get(type).new(:io => contents) if namespace
     end
+
+    def retrieve_latest(document_name)
+      blackboard = client.from_quiet_bucket
+      documents = inventory_summary_documents(blackboard, document_name)
+      get_latest_summary_report(documents)
+    end
+
+    private
+
+    def inventory_summary_documents(blackboard, doc_name)
+      doc_type = Regexp.new('\A' + doc_name + '(...)').freeze
```

Comment:
it makes sense to "freeze" this object since we want it to be immutable but since we're not consistent with it, I find it do be confusing here (why aren't we freezing documents on Line 56 for example)...